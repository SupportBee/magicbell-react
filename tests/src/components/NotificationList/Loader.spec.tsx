import { render, RenderResult } from '@testing-library/react';
import React from 'react';
import Loader from '../../../../src/components/NotificationList/Loader';

describe('components', () => {
  describe('NotificationList', () => {
    describe('Loader', () => {
      let view: RenderResult;

      beforeEach(() => {
        view = render(<Loader />);
      });

      afterEach(() => {
        view.unmount();
      });

      describe('render', () => {
        it('renders an animated loader', () => {
          expect(view.container).toMatchSnapshot();
        });
      });
    });
  });
});
