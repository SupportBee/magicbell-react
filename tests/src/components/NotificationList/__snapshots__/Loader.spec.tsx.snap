// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components NotificationList render renders an animated loader 1`] = `
.emotion-0 {
  color: inherit !important;
  font-size: 0.85em !important;
  padding: 0.5em !important;
  text-align: center !important;
}

<div>
  <div
    class="emotion-0"
  >
    Loading...
  </div>
</div>
`;
